tests/react-a11y-iframes/test.tsx.lint

```diff
@@ -0,0 +1,63 @@
+
+const SomeComponent = () => <iframe title="I'm a non empty title" src="http://someSource.com"/>;
+
+const SomeComponent = () => <iframe title={"hello there"} src="http://someSource.com"/>;
```

Comment:
Missing test case: `<iframe title={someVariable} ... />`

Reply:
@JoshuaKGoldberg added it quickly.
Just to clarify, you don't mean that this rule should check what `someVar` evaluates to, right?

Reply:
Correct, I think that's a bit out of scope. Though it might be doable 😉 this seems great as is.